src/box/vinyl.c

```diff
@@ -699,6 +699,11 @@ vinyl_space_check_index_def(struct space *space, struct index_def *index_def)
 			 "covering index");
 		return -1;
 	}
+	if (index_def->opts.layout != NULL) {
+		diag_set(ClientError, ER_UNSUPPORTED, "vinyl",
+			 "layout options");
+		return -1;
+	}
```

Comment:
Let's also provide layout info in index though `lbox_fillspace`. It can be tested however only in EE, like in case of `covers` option.

Reply:
Done.